### PR TITLE
Remove South Migrations

### LINKED ISSUE #116: Drop South Migrations
Since we've dropped support for earlier versions of Django, we should go ahead and drop the old South migrations as well.

### PR DESCRIPTION
**Pull request by mlavin**
Fixes #116. These files are no longer needed because South is no longer supported.

### CODE REVIEW

File: docs/releases.rst [hunk 1]
```diff
@@ -4,17 +4,34 @@ Release History
...
+
+The release removed the use of RequireJS for bundling the required assets
+and instead uses Browserify to create single required JS file.
+
+
+Features
+_________________
+
+- Added support for full screen edits
+
```
Comment (dchukhin) [inline docs/releases.rst]: Added support for Django 1.9?

File: README.rst [hunk 2]
```diff
@@ -26,8 +26,7 @@ Features
 Installation
 --------------------------------------
 
-django-scribbler requires Django >= 1.3 and Python >= 2.6. Starting with version v0.2
-there will be experimental Python 3 support (3.2+). Using Python 3 requires using Django >= 1.5.
+django-scribbler requires Django >= 1.8 and Python >= 2.7 or >= 3.3.
```
Comment (dchukhin) [inline README.rst]: note: Django 1.9 requires python >= 3.4

Comment (mlavin) [inline README.rst]: More of a Django issue documented in their supported version. I'd rather not repeat it here.

**Comment by mlavin** [comment]
All that's left here is some doc updates so I'm going to merge.

**Pull request merged**